### PR TITLE
Improve the documentation regarding contributions

### PR DESCRIPTION
**Pull request by TiagoMaiaL**
### Issue Link :link:

https://forums.swift.org/t/broken-unit-tests-for-the-ios-scheme/52371/5

### Goals :soccer:
- Document how to contribute to the project by sending pull requests. 
- Document how to configure the automated tests to be run.

### Implementation Details :construction:
I've changed the `CONTRIBUTING.md` file, adding a section about pull requests. I've also linked it from the project  `README.md`.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,6 +39,18 @@ When reporting issues, please fill out our issue template. The information the t
 
 You can triage issues which may include reproducing bug reports or asking for vital information, such as version numbers or reproduction instructions. If you would like to start triaging issues, one easy way to get started is to [subscribe to alamofire on CodeTriage](https://www.codetriage.com/alamofire/alamofire).
 
+## Submitting Pull Requests
+
+You can contribute by fixing bugs or adding new features. For larger code changes, we first recommend them to be discussed in the [project issues](https://github.com/Alamofire/Alamofire/issues) or in our [Swift Forums](https://forums.swift.org/c/related-projects/alamofire/). When submitting a pull request, please make sure your change doesn't break any automated tests (see [Automated Tests](#automated-tests) below).
```
Comment (jshier) [inline CONTRIBUTING.md:44]:
```suggestion
You can contribute by fixing bugs or adding new features. For larger code changes, we first recommend discussing them in our [GitHub issues](https://github.com/Alamofire/Alamofire/issues) or on our [Swift Forums](https://forums.swift.org/c/related-projects/alamofire/). When submitting a pull request, please add relevant tests and ensure your changes don't break any existing tests (see [Automated Tests](#automated-tests) below).
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -37,6 +39,18 @@ When reporting issues, please fill out our issue template. The information the t
...
+
+### Automated Tests
+
+The tests depend on [Firewalk](https://github.com/Alamofire/Firewalk), our HTTP test server. To run the automated tests, you first need to have the server running locally.
```
Comment (jshier) [inline CONTRIBUTING.md:48]:
```suggestion
Alamofire's tests depend on our [Firewalk](https://github.com/Alamofire/Firewalk) test server. To run the automated tests, you first need to have the server running locally. 
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -37,6 +39,18 @@ When reporting issues, please fill out our issue template. The information the t
...
+
+You can contribute by fixing bugs or adding new features. For larger code changes, we first recommend them to be discussed in the [project issues](https://github.com/Alamofire/Alamofire/issues) or in our [Swift Forums](https://forums.swift.org/c/related-projects/alamofire/). When submitting a pull request, please make sure your change doesn't break any automated tests (see [Automated Tests](#automated-tests) below).
+
+### Automated Tests
+
+The tests depend on [Firewalk](https://github.com/Alamofire/Firewalk), our HTTP test server. To run the automated tests, you first need to have the server running locally. 
+
+In your terminal, run the following commands:
+- To install Firewalk: `brew install alamofire/alamofire/firewalk`
+- To launch the server: `firewalk &`
```
Comment (jshier) [inline CONTRIBUTING.md:52]:
```suggestion
- To run and detach the server: `firewalk &`
- To stop the server, run `kill` and provide the pid output after launch.
```

File: README.md [hunk 4]
```diff
@@ -158,6 +158,10 @@ If you prefer not to use any of the aforementioned dependency managers, you can
 
   > The `Alamofire.framework` is automagically added as a target dependency, linked framework and embedded framework in a copy files build phase which is all you need to build on the simulator and a device.
 
+## Contributing
+
+Please read the instructions detailed in [CONTRIBUTING.md](./CONTRIBUTING.md).
```
Comment (jshier) [inline README.md:163]:
```suggestion
Before contributing to Alamofire, please read the instructions detailed our [contribution guide](https://github.com/Alamofire/Alamofire/blob/master/CONTRIBUTING.md).
```

**Review by jshier — CHANGES_REQUESTED** [review]
Looks pretty good, just some phrasing suggestions. Thanks for this!

**Comment by TiagoMaiaL** [comment]
Thanks for the correction and Improvements. I just applied them.

**Review by jshier — APPROVED** [review]
Thanks!

**Pull request merged**